### DISCUSSION TITLE
Discord or Slack Channel

### DISCUSSION THREAD

**Discussion by mertbakir** [valhalla/valhalla#3275] [General]
Hi, do we have a discord or slack channel?

**Comment by nilsnolde** [comment]
you're inside the only public channel there is. IMO project discussions should be topic-related to keep track so I personally like GH discussion better than e.g. slack channel

**Comment by kevinkreiser** [comment]
i have flirted with the idea of a real time chat system for the project in the past, i believe some people have had success with gitter. i think the problem will be one of bandwidth though. i already find it difficult to keep up with the async discussions here and in issues and prs, i think keeping up with them in a realtime format will be even more disappointing to the people coming there, plus its another channel of communication to keep my eye on. but i think the best reason given was what nils said, on github there is a much more searchable record for posterity and future folks lookign for answers or questions that may have been asked

**Comment by mertbakir** [comment]
I agree with you both in terms of project management but sometimes people may have a small question. For example, when you can't find something in the docs or don't know where to look for, or start with when you are a newbie. A chat channel would be helpful for those cases. I just wanted to know if there is an active platform like that. It looks like I'll be asking questions over gh discussions after searching of course.

**Reply by jverkoey** [reply]
Throwing my hat in as someone who'd love a more direct connection with the community as well :)

**Reply by kinkard** [reply]
Fortunately, this days, small questions about Valhalla could be addressed to GitHub Copilot. Here is the button:

<img width="398" height="94" alt="image" src="https://github.com/user-attachments/assets/6489f0b9-ff58-42fd-9270-03d0b5c5a1cf" />
